docker/DOCKER.md

```diff
@@ -0,0 +1,52 @@
+# Docker
+
+You can the application in a dev container or production/deployment container
+
+## Dev
+
+1. Open `env/development.env` and replace `API_KEY_GOES_HERE` with the youtube api key.
+
+```bash
+docker build -f docker/dev/Dockerfile -t opentogethertube-dev .
+```
+
+or just insert the api keys as a build argument for docker build
+
+```bash
+docker build -f docker/dev/Dockerfile -t opentogethertube-dev . --build-arg YOUTUBE_API_KEY=API_KEY_GOES_HERE
+```
+
+```bash
+docker run --name opentogethertube-dev -d -p 8080:8080 opentogethertube-dev
+
+# Run tests
+docker exec -it opentogethertube-dev npm test
+```
```

Comment:
Too many commands, not enough instructions. It reads more like a cheat sheet than a guide.